### PR TITLE
Improove documentation

### PR DESCRIPTION
**Pull request by Kaligule**
From what I learned today, this is the summary. Maybe someone should look over it, but it is better then nothing and definitelly would have helped me.

### CODE REVIEW

**Comment by Kaligule** [comment]
This adresses #46 and #47.

File: README.md [hunk 1]
```diff
@@ -39,18 +39,68 @@ Configuration
...
+
+### The main directory
+
+Everything Themer does happens in `.config/themer/`.
+
+In a running setup this directory will contain the following:
+
+- `templates`, a directory holding the templates for your all your
+  themes. This is where new themes are generated from.
+- as many themes (which are directories) as you want. These hold the
```
Comment (s-ol) [inline README.md]: "as you want" implies you create these, but usually they are generated for you, I would mention this to make it clear that `templates` and `current` are what you come in contact with and that the others are mostly for internal use

**Comment by s-ol** [comment]
looks pretty good overall.

**Review by s-ol — CHANGES_REQUESTED** [review]

**Comment by Kaligule** [comment]
I will change the pull request as soon as I have found out how to do it in github.

**Comment by Kaligule** [comment]
It seems my additional commit (which adresses your issues) has gone into the pullrequest without me doing anything. I am not sure what to think about that but I can see that it worked in this case. Any more changes?

**Comment by s-ol** [comment]
@Kaligule a PR is just a note to merge a branch on your repository with a branch on my repository (a *request* for me to *pull* your branch into my repo):

If you look at the top of this page, it says
>@Kaligule  wants to merge 6 commits into `s-ol:develop` from `Kaligule:develop`

As you can see the PR uses the branch labels, so when you update `Kaligule:develop` the PR changes as well. I'm not sure if Github supports this but theoretically the PR could be

> @Kaligule wants to merge 6 commits into `s-ol:develop` from e36527722bceee1a261f20f26d89ea6ec0828e9c

which would mean that when you change `develop` the PR stays the same. This is why usually you want to create a feature branch, like `update-documentation`, that is temporary so you can control it better on your end.

**Comment by s-ol** [comment]
Theres one minor word change I'm gonna add myself, but yeah this is fine.

**Comment by Kaligule** [comment]
So pull requests are about branches, not commits. I see my mistake. Thank you.

**Comment by s-ol** [comment]
Closed in e8934aeba54c3a4c0c460e0180c409bf097e8ae7 (squashed)

**Pull request closed without merging**